### DISCUSSION TITLE
⚡️ Xcode 15.3 will be default starting March 24

### DISCUSSION THREAD

**Discussion by helinanever** [codemagic-ci-cd/codemagic-docs#2407] [Announcements]
As of March 24, 2024, we are introducing the Xcode 15.3 build machine image as our new default. Additionally, Xcode latest will be switched to `15.3`.

Key points:

* If your Xcode version is set to `latest` or if no specific Xcode version is configured, then your builds will utilize the Xcode 15.3 image.
* Refer to the [build machine specs](https://docs.codemagic.io/specs/versions-macos/) for details on preinstalled software and available runtimes.

Action required:

* Please review your build configuration promptly to avoid unexpected behavior.
* You can always set a specific Xcode version in your build settings. Configure this in the Build section of the Flutter workflow editor or the environment section of codemagic.yaml.

If you have any questions or concerns, don't hesitate to get in touch with us.

**Comment by mikhail-tokarev** [comment]
❗ Please note: Apps uploaded to App Store Connect must be built with [Xcode 15](https://apps.apple.com/us/app/xcode/id497799835?mt=12) for [iOS 17](https://developer.apple.com/ios/), [iPadOS 17](https://developer.apple.com/ipados/), [tvOS 17](https://developer.apple.com/tvos/), or [watchOS 10](https://developer.apple.com/watchos/), starting **April 29, 2024**.


https://developer.apple.com/news/?id=fxu2qp7b
